### ISSUE TITLE
PROD - Application Performance

### ISSUE THREAD

**Issue by m-prodan** [bcgov/foi-flow#3770]
**Describe the bug in current situation**
Hearing user reports of application performance slowing around 2:45PM, on a daily occurrence.

So far this has been reported by the flex team, who have mentioned that this appears to be an issue for the whole team. Users report a lot of spinning wheels, especially for comments.

Further info from users - happens reliably around 2:45PM daily. Request itself can be leaded, but other tabs will load in perpetuity. Logging out and in appears to resolve it (Note from Matt: maybe this is a caching issue?)

**Link bug to the User Story**

**Impact of this bug**
Describe the impact, i.e. what the impact is, and number of users impacted. 

Slow performance can be frustrating for users - it is unclear how many users are impacted at this time, but the Intake team has intimated that it is a common issue across that team

**Chance of Occurring (high/medium/low/very low)**

**Pre Conditions: which Env, any pre-requesites or assumptions to execute steps?**

**Steps to Reproduce**

**Actual/ observed behaviour/ results**

**Expected behaviour**
A clear and concise description of what you expected to happen. Use the gherking language. 

**Screenshots/ Visual Reference/ Source**
If applicable, add screenshots to help explain your problem. You an use screengrab.

**Comment by m-prodan** [comment]
Updates - hearing users across the application reporting overall slowness of application. It is effecting things like assignments and some movement between screens. I have observed this myself now in PROD

**Comment by nkan-aot** [comment]
confirmed in a video call with Peter api request for comments and attachments are returning data. issue is strictly front end

**Comment by m-prodan** [comment]
@nkan-aot - I know for this one, you added a 'loading' feature to the tagged lists to attempt to fix the front end issue. I decided to try it out in TEST. One thing I noticed, is that the tag list no longer auto-filters as you enter a name? So its possible this bug fix may have introduced.

Worked it out with Loren - it appears to be impacting top-level comments - replies are filtering as expected:

![image.png](https://images.zenhubusercontent.com/6081a4c82f0c363dddb9c1a0/1a1445af-49d4-42e3-92d0-dd149d1d71e9)
